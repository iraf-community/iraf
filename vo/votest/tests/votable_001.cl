#
#  Print VOTable information

# Set the test description string.
votest.descr = "TABLES support: Print VOTable info"

# Convert the data$logical to a local path.
s1 = data_url // "/usno-b.xml"
s2 = "file://" // data_path // "/usno-b.xml"
s3 = "file:///localhost" // data_path // "/usno-b.xml"


fcache init

# Execute the test commands.
print ("\nLogical Path:  data$usno-b.xml")		# logical path
tinfo ("data$usno-b.xml")

print ("\nHTTP URI:  " // s1)				# remote http URI
tinfo (s1)

print ("\nFile URI:  " // s2)				# file URI
tinfo (s2)

print ("\nFile URI:  " // s3)				# file URI
tinfo (s3)
