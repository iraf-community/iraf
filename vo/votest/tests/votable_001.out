
Logical Path:  data$usno-b.xml
# data$usno-b.xml
  65 rows written to table
  16 columns defined
 188 header parameters written to table
 215 records allocated for header parameters
  16 space allocated for column descriptors
table type:  fits

HTTP URI:  http://iraf.noao.edu/votest//usno-b.xml
# http://iraf.noao.edu/votest//usno-b.xml
  65 rows written to table
  16 columns defined
 188 header parameters written to table
 215 records allocated for header parameters
  16 space allocated for column descriptors
table type:  fits

File URI:  file:///iraf/iraf/vo/votest/data//usno-b.xml
# file:///iraf/iraf/vo/votest/data//usno-b.xml
  65 rows written to table
  16 columns defined
 188 header parameters written to table
 215 records allocated for header parameters
  16 space allocated for column descriptors
table type:  fits

File URI:  file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml
# file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml
  65 rows written to table
  16 columns defined
 188 header parameters written to table
 215 records allocated for header parameters
  16 space allocated for column descriptors
table type:  fits
